util/backoff.go

```diff
@@ -9,6 +9,15 @@ import (
 	"time"
 )
 
+func init() {
+	// NOTE(sr): We don't need good random numbers here; it's used for jittering
+	// the backup timing a bit. But anyways, let's make it random enough; without
+	// a call to rand.Seed() we'd tge the same stream of numbers for each program
```

Comment:
```suggestion
	// a call to rand.Seed() we'd get the same stream of numbers for each program
```
[Nit] Small typo there.